docs/config/README.md

```diff
@@ -2082,6 +2083,45 @@ The module will be shown if any of the following conditions are met:
 format = "via [🏎  $version](red bold)"
 ```
 
+## Status
+
+The `status` module displays the exit code of the previous command.
+The module will be shown only if the exit code is not `0`.
+
```

Comment:
Let's add a tip here to make it explicitly obvious that this module is disabled by default
```suggestion

::: tip
                                                              
This module is disabled by default.
To enable it, set `disabled` to `false` in your configuration file.
:::

```